sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_file_client.py

```diff
@@ -545,8 +542,8 @@ def read_file(self, offset=None,   # type: Optional[int]
             The timeout parameter is expressed in seconds. This method may make
             multiple calls to the Azure service and the timeout will apply to
             each call individually.
-        :returns: downloaded data or the size of data written into the provided stream
-        :rtype: bytes or int
+        :returns: A streaming object (StorageStreamDownloader)
```

Comment:
Do you think we should add anything about readall here?

Reply:
I mean in the docstrings for download

Reply:
It's mentioned at the top of the docstring